internal/cmd/organization/role/role.go

```diff
@@ -12,8 +12,8 @@ import (
 func NewCmd(p *print.Printer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "role",
-		Short: "Provides functionality regarding organization roles",
-		Long:  "Provides functionality regarding organization roles.",
+		Short: "Provides functionality for organization roles",
+		Long:  "Provides functionality for organization roles.",
```

Comment:
Same here, maybe `Manage roles of an organization`?

Reply:
Or `Manage organization roles`